Source/GameBaseFramework/Public/Components/GBFUIDialogManagerComponent.h

```diff
@@ -69,12 +69,18 @@ class GAMEBASEFRAMEWORK_API UGBFUIDialogManagerComponent : public UActorComponen
 
     bool IsDisplayingDialog() const;
 
-    UFUNCTION( BlueprintCallable )
-    void InitializeMainUI( const TSubclassOf< UUserWidget > & main_ui_class );
+    UFUNCTION( BlueprintCallable, DisplayName = "InitializeMainUI" )
+    void InitializeMainUIWithClass( const TSubclassOf< UUserWidget > & main_ui_class );
+
+    UFUNCTION( BlueprintCallable, DisplayName = "InitializeMainUI" )
+    void InitializeMainUI();
 
     UFUNCTION( BlueprintCallable )
     void ShowMainUI();
 
+    UFUNCTION( BlueprintCallable )

```

Comment:
It does not need to be BlueprintCallable. Since it acts on whether or not the action is done in BeginPlay, calling it in BP would be too late because Beginplay would already have happened.

That function is mainly to be used in the constructors of the player controller